src/Development/IDE/Core/Rules.hs

```diff
@@ -675,6 +675,7 @@ getModSummaryRule = defineEarlyCutoff $ \GetModSummary f -> do
 
 getModIfaceRule :: Rules ()
 getModIfaceRule = define $ \GetModIface f -> do
+#if MIN_GHC_API_VERSION(8,6,0) && !defined(GHC_LIB)
```

Comment:
What’s up with this? What changed here?

Reply:
The benchmark tests revealed that `getDefinition` was broken in GHC 8.4. This fixes it. I can add more detail if needed.

Reply:
No need to add anything more, I was just curious how this was related to this PR. Thanks!